i18n/ja/docusaurus-plugin-content-docs/current/introduction/what-is-serverless-caching.md

```diff
@@ -1,106 +1,107 @@
 
-As Ben Kehoe notes in the article linked above, "serverlessness" is a spectrum, and you want to be using services that tend to have more of the qualities above. Additionally, some services have flexibility in how you use them, but you should try to use them in a more serverless way.
+最後に、サーバーレスは従量課金のスキーマです。サービスが、その下で動くリソースよりも抽象化されていてキャパシティ指向なので、支払うお金と受け取る価値にほぼ直接的な関連性を見いだせるでしょう。これは、例えば AWS Lambda、Amazon API Gateway、または Amazon SQS の様な従量課金のシステムを意味します。他にも、Amazon DynamoDB や Amazon Kinesis の様な柔軟にスケールアップやダウンできるシステムを意味することもあります。サーバーレスサービスでは、ピーク時の負荷に備えてオーバープロビジョニングして、ほとんどの時間はインスタンスが低いリソース利用率にある様な事は必要はありません。
 
-### How Momento Serverless Cache fits the traditional definition of serverless
+Ben Kehoe が上述の記事で言っている様に、"サーバーレス"とはスペクトラムであり、上記の属性をより多く持っているサービスを使いたくなるでしょう。加えて、いくつかのサービスは利用方法に柔軟性がありますが、よりサーバーレス的な使い方ができる方法を試してみるべきです。
 
-Momento Serverless Cache fits perfectly under this traditional definition of serverless.
+### 古典的なサーバーレスの定義に Momento がどの様に適合するか
 
-First, Momento Serverless Cache is a managed service. You won't be installing software, managing failovers, or upgrading versions. This is handled for you behind the scenes so you can focus on building and maintaining the core parts of your application.
+Momento Cache はこの古典的なサーバーレスの定義に完璧に適合しています。
 
-Second, Momento Serverless Cache abstracts the decisions around cache management from you. You don't have to specify the type and number of instances in your cache, nor do you have to think about the maximum memory size for your cache. Momento is purpose-built for the cloud and takes advantage of the elasticity and scalability of modern cloud infrastructure. You can store as much data as you need in your cache, and Momento will handle it seamlessly.
+第一に、Momento Cache はマネージドサービスです。ソフトウェアをインストールしたり、フェイルオーバーを管理したり、バージョンを更新したりする必要はありません。これらは裏で適切に処理されているので、皆さんはご自身のアプリケーションの中心となる場所の開発と運用に専念することができます。
 
-Finally, Momento Serverless Cache has a pay-for-value pricing model. As mentioned above, you aren't selecting a specific cache and cluster size in advance, regardless of whether you use it. Momento charges based on the amount of data stored and the requests made against your cache. With this, you are in control of your pricing, and changes you make to your application have a direct impact on your bill.
+第二に、Momento Cache はキャッシュ管理に関する設定を抽象化しています。キャッシュに必要なインスタンスの数を指定することも、最大メモリサイズを考えることも必要ありません。Momento はクラウドに特化して作られていて、現代的なクラウド基盤の弾力性やスケーラビリティを最大活用しています。皆さんはキャッシュに必要なデータをどれだけでも保存することができて、Momento がそれをシームレスに取り扱います。
 
-Under this first definition of serverless, Memento Serverless Cache is the cache that fits best in the serverless ecosystem.
+最後に、Momento Cache は従量課金のモデルです。上述の様に、利用の有無に関係なく、事前に特定のキャッシュの種類やクラスターサイズを選択することはありません。Momento はキャッシュに保存されたデータ量と行われたリクエストに対して課金をします。これによって、料金は皆さんの支配下にあって、アプリケーションの変更が直接的に請求金額に影響を与えます。
```

Comment:
nit: "Momento charges based on the amount of data stored" could miss-lead as its cost dimension is transferred data volume.

Reply:
Yeah, this is old wording. Let me look at the page this is based on and we can update it.

Reply:
This should read "Momento charges for data transferred in/out of Momento Cache and Momento Topics services. Everything else is included."

Reply:
If it is possible, please add a link to the pricing page for more information. https://docs.momentohq.com/manage/pricing

Reply:
Looks good. Will update both EN and JA.

Reply:
Updated.